astropy/convolution/convolve.py

```diff
@@ -869,3 +870,28 @@ def convolve_models(model, kernel, mode='convolve_fft', **kwargs):
         raise ValueError(f'Mode {mode} is not supported.')
 
     return CompoundModel(mode, model, kernel)
+
+
+def convolve_models_fft(model, kernel, bounding_box, resolution, cache=True, **kwargs):
```

Comment:
Please describe ``bounding_box``, ``resolution``, and ``cache`` in the Parameters section of the docstring.

Reply:
Added in 6a326a3.